bip-0047.mediawiki

```diff
@@ -338,7 +338,7 @@ In order to use Bitmessage notification, the recipient must have a Bitmessage cl
 
 ==Version 2==
 
-Version 2 payment codes behave identifically to version 1 payment codes, except as modified below.
+Version 2 payment codes behave identically to version 1 payment codes, except as modified below.
```

Comment:
I think "identific" is an older and relatively rare word that means "identical in action" and so isn't necessarily in need of being changed.